src/lib/MultiCombobox/MultiCombobox.svelte

```
@@ -0,0 +1,465 @@
+    // are hidden and a limit message is shown instead (a custom action stays).
+    ...(atLimit ? [] : filtered.map((item): Row => ({ kind: 'item', item }))),
+    ...(showCreate ? [{ kind: 'create' } as Row] : []),
+    ...(action ? [{ kind: 'action' } as Row] : [])
+  ]);
+
+  // Keep the highlight within range as the row set changes.
+  $effect(() => {
+    if (highlighted > rows.length - 1) {
+      highlighted = rows.length > 0 ? rows.length - 1 : 0;
+    }
+  });
```

ESLint

[error] $effect is not allowed. Use a different reactive pattern instead.

(no-restricted-syntax)